### PR TITLE
Issue 76 infra microservices

### PR DESCRIPTION
**Pull request by gloriajw**
Converted all processes as apps able to be dispatched. Added a unit test just to make sure imports and app init works. Looks like it;s working.

### CODE REVIEW

File: mqtt_client/mqtt_client.py [hunk 1]
```diff
@@ -131,6 +131,7 @@ def client_shutdown(self):
     client_id = str(uuid.uuid4())
     topic = "event"
 
+    import pdb;pdb.set_trace()
```
Comment (paul-rohrlach) [inline mqtt_client/mqtt_client.py:134]: We should probably pull this out before merging

**Review by paul-rohrlach — CHANGES_REQUESTED** [review]
Love it!

So much less code 😍 

Just the one (pedantic) note about removing debug code before we merge to `main`

**Review by paul-rohrlach — APPROVED** [review]
Let me get that for you!

File: mqtt_client/mqtt_client.py (hunk 1 above)
Comment (gloriajw) [reply mqtt_client/mqtt_client.py]: I missed that, thank you! That got committed by accident. Removing...

Comment (gloriajw) [reply mqtt_client/mqtt_client.py]: Done!

Comment (gloriajw) [reply mqtt_client/mqtt_client.py]: I just saw your comment, thanks!

Comment (paul-rohrlach) [reply mqtt_client/mqtt_client.py]: I do it *all* the time - It's why I notice it haha.

**Comment by vickis** [comment]
:)

**Pull request merged**